src/frontend/src/pages/Result.tsx

```diff
@@ -8,44 +8,40 @@ import { useNavigate, useSearchParams } from "react-router-dom";
 import { LatLngExpression } from "leaflet";
 import { motion } from "framer-motion";
 import { PhotoCamera, ArrowBack } from "@mui/icons-material";
-import React, { useEffect, useState } from "react";
+import React, { useState } from "react";
 import MapPreview from "../components/MapPreview";
-import { getPath } from "../services/Graph";
-import { IGraphLocation } from "../types";
-import Lottie from "lottie-react";
-
-import loadingAnimation from "../assets/animations/loadingMap.json";
-
-// function that converts the path to objects that can be used by the map
-function pathToPoints(path: IGraphLocation[]): LatLngExpression[] {
-  return path.map((p) => ({ lat: p.latitude, lng: p.longitude }));
-}
 
 function Result() {
-  // Navigation
   const navigate = useNavigate();
 
-  // Search params
   let [searchParams, setSearchParams] = useSearchParams();
-  let [pathID, setPathID] = useState(searchParams.get("pathID"));
-
-  const [path, setPath] = useState<IGraphLocation[]>([]);
-  const [points, setPoints] = useState<LatLngExpression[]>([]);
-
-  // react hook that loads the path from the server
-  useEffect(() => {
-    async function getPathAsync() {
-      const path = await getPath(pathID || "");
-      setPath(path);
-    }
-    getPathAsync();
-  }, []);
+  let [originLat, setOriginLat] = useState(searchParams.get("originLat"));
+  let [originLon, setOriginLon] = useState(searchParams.get("originLon"));
```

Comment:
It seems like there may be a problem with this page specifically. These changes are replacing the master version, which is the last updated one.

Reply:
This is just a subsection but the problem seems to be present with the entire "Results" file.